tsinfer/inference.py

```diff
@@ -1982,6 +1988,9 @@ def post_process(
     :param bool split_mrca: If ``True`` (default) and the oldest node is the only
         parent to a single "grand MRCA", split the grand MRCA into separate nodes
         (see above). If ``False`` do not attempt to identify or split a grand MRCA.
+    :param bool keep_flanks: If ``True``, output regions of topology inferred before
+        the first site and after the last site. Otherwise (default) keep only the
+        inferred topology between the first and last sites.
     :return: The post-processed tree sequence.
     :rtype: tskit.TreeSequence
     """
```

Comment:
Missing default assignment for ``trim_flanks`` here, ``trim_flanks = True if trim_flanks is None else trim_flanks``.